### ISSUE TITLE
Support Standback

### ISSUE THREAD

**Issue by Koxiaet** [jplatte/caniuse.rs#3]
[Standback](https://crates.io/crates/standback) allows for newer Rust std APIs to be used from older versions of Rust. Features should additionally have a standback badge if they are not supported in the version of Rust, but are by Standback.

**Comment by jplatte** [comment]
I don't think this is something I want *right now* when standback is still very new and hasn't proven itself, but I'm definitely open to adding this in the future.
